[2025-03-05T17:13:58.153+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-05T17:13:58.177+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: task_etl_fire_incident_using_bash.task_etl_fire_incident_using_bash manual__2025-03-05T15:32:20.490427+00:00 [queued]>
[2025-03-05T17:13:58.192+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: task_etl_fire_incident_using_bash.task_etl_fire_incident_using_bash manual__2025-03-05T15:32:20.490427+00:00 [queued]>
[2025-03-05T17:13:58.193+0000] {taskinstance.py:2867} INFO - Starting attempt 2 of 2
[2025-03-05T17:13:58.212+0000] {taskinstance.py:2890} INFO - Executing <Task(BashOperator): task_etl_fire_incident_using_bash> on 2025-03-05 15:32:20.490427+00:00
[2025-03-05T17:13:58.224+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'task_etl_fire_incident_using_bash', 'task_etl_fire_incident_using_bash', 'manual__2025-03-05T15:32:20.490427+00:00', '--job-id', '240', '--raw', '--subdir', 'DAGS_FOLDER/fire_incidents_dag.py', '--cfg-path', '/tmp/tmpf_qnac63']
[2025-03-05T17:13:58.225+0000] {standard_task_runner.py:105} INFO - Job 240: Subtask task_etl_fire_incident_using_bash
[2025-03-05T17:13:58.231+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=60) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-03-05T17:13:58.232+0000] {standard_task_runner.py:72} INFO - Started process 62 to run task
[2025-03-05T17:13:58.741+0000] {task_command.py:467} INFO - Running <TaskInstance: task_etl_fire_incident_using_bash.task_etl_fire_incident_using_bash manual__2025-03-05T15:32:20.490427+00:00 [running]> on host 414e96ca1b1a
[2025-03-05T17:13:58.878+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='***@***.com' AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='task_etl_fire_incident_using_bash' AIRFLOW_CTX_TASK_ID='task_etl_fire_incident_using_bash' AIRFLOW_CTX_EXECUTION_DATE='2025-03-05T15:32:20.490427+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-03-05T15:32:20.490427+00:00'
[2025-03-05T17:13:58.880+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-03-05T17:13:58.881+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-03-05T17:13:58.882+0000] {logging_mixin.py:190} INFO - Current task name:task_etl_fire_incident_using_bash state:running start_date:2025-03-05 17:13:58.178325+00:00
[2025-03-05T17:13:58.882+0000] {logging_mixin.py:190} INFO - Dag name:task_etl_fire_incident_using_bash and current dag run status:running
[2025-03-05T17:13:58.882+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-03-05T17:13:58.884+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-03-05T17:13:58.886+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', 'echo "Hello, world!" && ls -l && pwd && whoami']
[2025-03-05T17:13:58.908+0000] {subprocess.py:99} INFO - Output:
[2025-03-05T17:13:58.911+0000] {subprocess.py:106} INFO - Hello, world!
[2025-03-05T17:13:58.926+0000] {subprocess.py:106} INFO - total 0
[2025-03-05T17:13:58.927+0000] {subprocess.py:106} INFO - /tmp/***tmpwqoi7bqd
[2025-03-05T17:13:58.928+0000] {subprocess.py:106} INFO - ***
[2025-03-05T17:13:58.928+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2025-03-05T17:13:58.970+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-03-05T17:13:58.971+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=task_etl_fire_incident_using_bash, task_id=task_etl_fire_incident_using_bash, run_id=manual__2025-03-05T15:32:20.490427+00:00, execution_date=20250305T153220, start_date=20250305T171358, end_date=20250305T171358
[2025-03-05T17:13:59.108+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2025-03-05T17:13:59.109+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2025-03-05T17:13:59.109+0000] {logging_mixin.py:190} INFO - Dag name:task_etl_fire_incident_using_bash queued_at:2025-03-05 15:32:20.507682+00:00
[2025-03-05T17:13:59.110+0000] {logging_mixin.py:190} INFO - Task hostname:414e96ca1b1a operator:BashOperator
[2025-03-05T17:13:59.132+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-05T17:13:59.445+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-03-05T17:13:59.448+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
